src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,45 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val englishGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello world"
+            }
+            override fun greetSomeone(someone: String): String {
+                return "Hello $someone"
+            }
+        }
+
+        val frenchGreeting  = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Salut tout le monde"
+            }
+            override fun greetSomeone(someone: String): String {
+                return "Salut $someone"
+            }
+        }
+
+        val spanishGreeting  = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hola, mundo"
+            }
+            override fun greetSomeone(someone: String): String {
+                return "Hola, $someone"
+            }
+        }
+
+        val greetings = mutableListOf<String>()
+
+        greetings.add(englishGreeting.greet())
```

Comment:
Code duplication is unflexible solution.
Try to ask yourself during coding "What if there will be 1M items?"